src/main.ts

```diff
@@ -156,11 +156,11 @@ export async function runAction(): Promise<any> {
     if (
       !Object.values(constants.EndStates).includes(executionGraph["status"])
     ) {
-      core.setFailed(`Execution graph ${executionGraphId} has timed out.`)
+      core.info(`Execution graph ${executionGraphId} has timed out.`)
     } else {
       if (executionGraph["status"] !== constants.EndStates.SUCCEEDED) {
         displayErrorExecutionGraph(executionGraph)
-        core.setFailed(
+        core.info(
```

Comment:
If you don't set a `failedMessage` value here, then the if condition at the bottom won't pass and this is a regression.